src/errors/DJSError.js

```diff
@@ -15,7 +15,7 @@ function makeDiscordjsError(Base) {
     constructor(key, ...args) {
       super(message(key, args));
       this[kCode] = key;
-      if (Error.captureStackTrace) Error.captureStackTrace(this, DiscordjsError);
+      Error.captureStackTrace?.(this, DiscordjsError);
```

Comment:
The conditional was made because [Error.captureStackTrace](https://nodejs.org/api/errors.html#errors_error_capturestacktrace_targetobject_constructoropt) is a non-standard feature implemented only by V8.

Since we do not longer use webpack, we can remove said conditional.

```suggestion
      Error.captureStackTrace(this, DiscordjsError);
```